import { AxiosError } from 'axios';
export declare const handleError: (error: AxiosError) => void;
